src/index.html

```diff
@@ -3,11 +3,100 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <ul class="stars__list">
+        <li class="stars__star">
+        </li>
+        <li class="stars__star">
+        </li>
+        <li class="stars__star">
+        </li>
+        <li class="stars__star">
+        </li>
+        <li class="stars__star">
+        </li>
+      </ul>
```

Comment:
Let's not complicate the things.
Just make 5 `div` elements without the lists.
This way HTML and styles will be more simple.

Reply:
thanks for help!